frontend/src/lib/k8s/api/v2/hooks.ts

```diff
@@ -275,3 +283,77 @@ export function useKubeObjectList<T extends KubeObjectClass>({
     },
   };
 }
+
+/**
+ * Returns a combined list of Kubernetes objects and watches for changes from the clusters given.
+ */
+export function useKubeObjectLists<T extends KubeObjectClass>({
```

Comment:
I had actually thought that we could adapt the useKubeObjectList and pass a new `clusters` arg. But then we'd have to use `useQueries` if we want to maintain a cache of each cluster's results independently and maybe that adds complexity. So maybe we can use this new function.
I'd like @sniok 's opinion here.

Reply:
I removed useKubeObjectLists, and now just useKubeObjectList is exported. The multi cluster version is internal.